.ci/scripts/utils.sh

```diff
@@ -62,10 +62,38 @@ install_pytorch_and_domains() {
   git checkout "${TORCH_VERSION}"
   git submodule update --init --recursive
```

Comment:
can move this command (cloning all submodules) when we haven't found the cache entry?